### ISSUE TITLE
test | block google analytics when running tests

### ISSUE THREAD

**Issue by NoamGaash** [hasadna/open-bus-map-search#829]
When running playwright test, let's use page.route to abort all requests to google analytics

**Comment by NoamGaash** [comment]
@YaelChen I think there are some domains that we would like to block, while avoid blocking unrelated urls
Using the Chrome DevTool can help determine which URLs should be blocked:
![image](https://github.com/user-attachments/assets/dcc4e708-6d9d-46ce-a98c-64a50643a8f0)
And the test you've added that clicks this link is expected to pass despite of this change:
![image](https://github.com/user-attachments/assets/0b6d9fa4-e6c0-46b3-bef4-dd38bf86cf5b)

Thank you! :pray:

**Comment by YaelChen** [comment]
I see the logic 😅
Thanks! I'll need to investigate some more for this,
So I'm not assigning yet so that someone else will be able to take it :)

**Comment by NoamGaash** [comment]
I believe that the command `await page.route('**/*google-analytics*', route => route.abort());` use a wrong pattern.
something like:
```
page.route(/google-analytics\.com|googletagmanager\.com/, route => route.abort());
```
or
```
page.route(/^https?:\/\/(google-analytics|googletagmanager).com/, route => route.abort());
```

**Comment by NoamGaash** [comment]
closed by @LeoBonjo  in #951 - Thank you very much!
